network/nebula/nebula.yaml

```diff
@@ -0,0 +1,47 @@
+name: nebula
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+  - configuration: true
+container:
+  entrypoint: /usr/local/bin/nebula
+  args: 
+    - -config 
+    - /usr/local/etc/nebula/config.yml 
+  environment:
+  - PATH=/sbin:/usr/local/bin
+  security:
+    writeableRootfs: false
+    writeableSysfs: true
```

Comment:
is this needed?

Reply:
I'm pretty sure it needs Sysfs, since it uses tun. It shouldn't need to write to root.

If you mean the PATH, I just took that from Tailscale, I could probably remove it.